adaptations/test_Clavia_Nord_Lead.py

```
@@ -0,0 +1,217 @@
+def vlq(value):
+    result = [value & 127]
+    while value > 127:
+        value >>= 7
+        result.insert(0, (value & 127) | 128)
+    return bytes(result)
+
+
+EOT = b"\x00\xff\x2f\x00"
+
+
+def smf(*tracks, format=0):
```

Ruff (0.16.3)

[error] Function argument `format` is shadowing a Python builtin

(A002)